models/userStatus.js

```diff
@@ -23,6 +23,7 @@ const { userState } = require("../constants/userStatus");
 const discordRoleModel = firestore.collection("discord-roles");
 const memberRoleModel = firestore.collection("member-group-roles");
 const usersCollection = firestore.collection("users");
+const config = require("config");
```

Comment:
why this change?

Reply:
While running the test, I got an error `config is not defined`. `config.get` is used in `userStatus.js` without importing the config file and hence the error. You can check the same here: https://github.com/Real-Dev-Squad/website-backend/blob/develop/models/userStatus.js#L26

Reply:
config is a global variable. How it it giving error in this file.
I am sure that config is being used in multiple places.

Reply:
I am not sure why it is giving error for this file. I only made this change to get rid of the errors that I encountered while running tests.
Let me know about the next steps.

Reply:
@pankajjs Can I resolve the conversation?